scripts/article1_init.gml

```diff
@@ -7,4 +7,7 @@ tethered_id = noone;
 chain_length = player_id.chain_length * 16;
 chain_segments = player_id.chain_segments;
 owner_true_y = 0;
-tethered_true_y = 0;
\ No newline at end of file
+tethered_true_y = 0;
+chain_link_sprite = sprite_get("chain_link");
+chain_link_sprite2 = sprite_get("chain_link2");
```

Comment:
two sprites here because we set the origin in load.gml differently for them. chain link 1 works from left to right while chain link 2 works right to left